evaluation/integration_tests/tests/t01_fix_simple_typo.py

```diff
@@ -8,7 +8,7 @@
 
 
 class Test(BaseIntegrationTest):
-    INSTRUCTION = 'Fix typos in bad.txt.'
+    INSTRUCTION = 'Fix typos in /workspace/bad.txt.'
```

Comment:
this seems like an anti-pattern--end users are not going to specify absolute paths...

Reply:
just fixed in the most recent commit